### DISCUSSION TITLE
Can't find an article for `view-transition-class` property

### DISCUSSION THREAD

**Discussion by cssinate** [mdn/mdn-community#692] [Content] [answered]
The [spec for the view-transitions-2 module](https://www.w3.org/TR/css-view-transitions-2/) includes the [view-transition-class property](https://www.w3.org/TR/css-view-transitions-2/#view-transition-class-prop). I can't find any references to this property in MDN. I'm happy to make the contribution, but my understanding of the contribution guidelines are that the content should appear on the contributors task board. If this gets added, I'll be happy to pick it up and run with it.

**Comment by Josh-Cena** [comment, marked as answer]
Hi @cssinate, is this implemented anywhere? We only document if it's in at least one browser.

Also, you can directly send an issue to https://github.com/mdn/content. If it gets accepted (that is, triaged without closing) you can go ahead and send the PR.

**Comment by cssinate** [comment]
Yes, it's implemented in Chromium. I'll create the issue. Thanks!
